### PR TITLE
Moved content from js-projects repo

### PR DESCRIPTION
**Pull request by adamblanchard**
Since we're now going to officially use the js-projects in replacement of the existing homework exercises, it makes sense to me to move it into the actual javascript curriculum repo.

I'll make further changes on top of this in a separate PR.

### CODE REVIEW

File: .DS_Store
Comment (vdaguenet) [inline .DS_Store:1]: We should avoid tracking .DS_Store files.

**Review by vdaguenet — APPROVED** [review]

**Review by jason-vasilev — APPROVED** [review]

File: homework-projects/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- You will apply what you learned every week and understand how each new method can be implemented into an actual product.
+- You will build a solid project for your portfolio that you can showcase on your CV and in future tech interviews!
+- You will gain experience presenting your project and code in a 'mini-tech interview' at the end of JS3.
+
+<br/>
+
+# HOW
+
+<br/>
+First, you will pick a project idea (see the "[projects](url)" folder). Each week, you will have specific tasks that you have to deliver. Those tasks are provided in your chosen project description. The tasks are designed to help you practice the main learning points throughout the JS2 and JS3 modules:
```
Comment (magdazelena) [inline homework-projects/README.md]: is the `url` here a stub for purpose?

**Review by magdazelena — APPROVED** [review]
I caught one (possible) typo, rest looks great :)

File: .DS_Store
Comment (adamblanchard) [reply .DS_Store:1]: My bad, will remove!

File: homework-projects/README.md (hunk 1 above)
Comment (adamblanchard) [reply homework-projects/README.md]: Ah yes, good catch - i'll fix it! Thanks!

**Pull request merged**